packages/react-charting/README.md

```diff
@@ -61,7 +61,7 @@ A comprehensive contributor guide is available in our internal [wiki](https://ak
 
 ## Testing
 
-![Static Badge](https://img.shields.io/badge/coverage-87%25-brightgreen)
+![Test Coverage Badge](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Ffluentui-charting-contrib%2Ftest-coverage-artifacts%2FubuntuCoverage.json&query=%24.statementCoverage&suffix=%25&label=coverage&color=brightgreen)
```

Comment:
what is the fallback if dynamic file fails to load

Reply:
In such cases... a default invalid badge is displayed by the provider

Reply:
can you show example

Reply:
In case of key or file not found
![image](https://github.com/microsoft/fluentui/assets/35366351/e836f42e-16d4-475e-be0b-9b0fa539a7bb)
Fetch Failure
![image](https://github.com/microsoft/fluentui/assets/35366351/deceaf97-3d43-46fb-b242-a3da715c3462)

Reply:
This is plain markdown rendering. Markdown doesn't directly support handling image fetching errors or displaying fallback images.

Reply:
ok